.github/workflows/ci-pytorch-test-tpu.yml

```diff
@@ -0,0 +1,131 @@
+        python-version: ${{ env.PYTHON_VER }}
+
+    - name: Checkout ml-testing-accelerators
+      run: |
+       git clone https://github.com/GoogleCloudPlatform/ml-testing-accelerators.git
+       cd ml-testing-accelerators
+       git fetch origin 5e88ac24f631c27045e62f0e8d5dfcf34e425e25:stable
+       git checkout stable
+
+    - uses: actions/setup-go@v3
+      with:
+        go-version: '1.19'
+
+    - name: Install jsonnet
+      run: go install github.com/google/go-jsonnet/cmd/jsonnet@latest
+
+    - name: Update jsonnet
+      env:
+        XLA_VER: 1.12
+        PR_NUMBER: ${{ github.event.pull_request.number }}
+        SHA: ${{ github.event.pull_request.head.sha }}
+      run: |
+       python -c "fname = 'dockers/tpu-tests/tpu_test_cases.jsonnet' ; data = open(fname).read().replace('{PYTORCH_VERSION}', '$XLA_VER')
+       data = data.replace('{PYTHON_VERSION}', '$PYTHON_VER').replace('{PR_NUMBER}', '$PR_NUMBER').replace('{SHA}', '$SHA') ; open(fname, 'w').write(data)"
+       cat dockers/tpu-tests/tpu_test_cases.jsonnet
+      shell: bash
+
+    # https://docs.github.com/en/actions/deployment/deploying-to-your-cloud-provider/deploying-to-google-kubernetes-engine
+    - uses: google-github-actions/get-gke-credentials@fb08709ba27618c31c09e014e1d8364b02e5042e
+      with:
+        cluster_name: ${{ env.GKE_CLUSTER }}
+        location: ${{ env.GKE_ZONE }}
+        credentials: ${{ secrets.GKE_SA_KEY_BASE64 }}
+
+    - name: Deploy cluster
+      run: |
+       export PATH=$PATH:$HOME/go/bin
+       job_name=$(jsonnet -J ml-testing-accelerators/ dockers/tpu-tests/tpu_test_cases.jsonnet | kubectl create -f -)
+       job_name=${job_name#job.batch/}
+       job_name=${job_name% created}
+       pod_name=$(kubectl get po -l controller-uid=`kubectl get job $job_name -o "jsonpath={.metadata.labels.controller-uid}"` | awk 'match($0,!/NAME/) {print $1}')
+       echo "GKE pod name: $pod_name"
+       echo "Waiting on kubernetes job: $job_name"
+       status_code=2 &&
+       # Check on the job periodically. Set the status code depending on what happened to the job in Kubernetes.
+       printf "Waiting for job to finish: "
+       while true; do
+         if kubectl get jobs $job_name -o jsonpath='Failed:{.status.failed}' | grep "Failed:1"; then
+           status_code=1 && break;
+         elif kubectl get jobs $job_name -o jsonpath='Succeeded:{.status.succeeded}' | grep "Succeeded:1"; then
+           status_code=0 && break;
+         else
+           printf ".";
+         fi;
+         sleep 5;
+       done
+       echo "Done waiting. Job status code: $status_code"
+       kubectl logs -f $pod_name --container=train > /tmp/full_output.txt
+       if grep -q '<?xml version="1.0" ?>' /tmp/full_output.txt; then
+         csplit /tmp/full_output.txt '/<?xml version="1.0" ?>/';
+       else
+         mv /tmp/full_output.txt xx00;
+       fi
+       # First portion is the test logs.
+       cat xx00 && echo "Done with log retrieval attempt."
```

Comment:
This `xx00` file is empty so `coverage report` fails below with `No data to report.`

Any ideas @akihironitta @Borda?

Reply:
How about the `/tmp/full_output.txt` above? Is it also empty?
cc: @kaushikb11